telegram/constants.py

```diff
@@ -277,6 +279,22 @@ class ChatType(StringEnum):
     """:obj:`str`: A :class:`telegram.Chat` that is a channel."""
 
 
+class CustomEmojiIdentifiersLimit(IntEnum):
```

Comment:
```suggestion
class CustomEmojiStickerLimit(IntEnum):
```
? that's a bit more generic. gives us the option to use it for something else than just identifiers …

Reply:
Sure.